### PR TITLE
Adjust contribtors and add acknowledgments.

### PR DESCRIPTION
**Pull request by xorrkaz**
Turn the existing contributors section into an acknowledgments section and add a new contributors section recognizing those who made substantial contributions who are not listed as authors.

### CODE REVIEW

**Comment by Reshad-Rahman** [comment]
LGTM

File: yang-semver/draft-ietf-netmod-yang-semver.xml [hunk 1]
```diff
@@ -784,6 +799,8 @@ module ietf-yang-semver {
 .  We would like thank both
    Anees Shaikh and Rob Shakir for their input into this problem
    space.</t>
```
[the remark this answers is not in the document]
Comment (xorrkaz) [reply yang-semver/draft-ietf-netmod-yang-semver.xml:801]: It's virtually the same sentence here as in module versioning.  However, here we have added a link to OpenConfig semver.  I don't see what is missing.

[the remark this answers is not in the document]
Comment (xorrkaz) [reply yang-semver/draft-ietf-netmod-yang-semver.xml:801]: Ah, I see why you think it's less.  GH is only showing the diff.  If you expand the other lines you'll see the text is virtually the same as module versioning.

[the remark this answers is not in the document]
Comment (jsterne) [reply yang-semver/draft-ietf-netmod-yang-semver.xml:801]: Ahhh yes. Didn't realize that. Looks good then.

**Pull request merged**